### DISCUSSION TITLE
Git duet commit issue - contributions graph

### DISCUSSION THREAD

**Discussion by nafizcodes** [community/community#144011] [Other Feature Feedback, Questions, & Ideas]
### Git duet contributions graph

I have made some contributions which show up on my calendar but I don’t see any contributions in our repo contributors list for some reason. I am thinking its an issue due to the git-duet commit. So I have been using git-duet and I think I pushed the changes only myself excluding the other person but it was still on git duet. So it showed both the author and committer only me. Can you please help how I can make the contributions show the graph ?

<img width="602" alt="1" src="https://github.com/user-attachments/assets/d621474a-42e9-4fc0-9f2c-a99e24a7ae67">
<img width="296" alt="2" src="https://github.com/user-attachments/assets/5ff2e9a8-3392-4591-bf99-d71ea8e56682">
<img width="228" alt="3" src="https://github.com/user-attachments/assets/a037165d-46ed-4aeb-9999-5c0da42e201f">
<img width="425" alt="image" src="https://github.com/user-attachments/assets/6b93ecc4-2d3e-46ff-abad-00e514067c1c">

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
